PET_Clinic/PET_INSTRUMENTATION.md

```diff
@@ -0,0 +1,119 @@
+# Instrumentation configuration
+The goal in this part of the workshop is to create a basic instrumentation for our sample application _Spring Petclinic Microservices_.
+
+The prerequisite for this part is that you have finished the [setup part](PET_SETUP.md) of the workshop. We also assume that you have your _Spring Petclinic Microservices_ up and running with the five inspectIT agents.
+
+## Configuration perspective
+For all the instrumentation configurations you need to switch to the ![Configuration perspective](../images/compass.png?raw=true) *Configuration perspective*. As in all Eclipse-based application, perspective bar is available on the top-right of your application window.
+
+## Environment creation
+The first thing to do is to create a new environment that will be used by the inspectIT agents that runs within the sample application. From the ![Configuration Manager](../images/compass.png?raw=true) **Configuration Manager** view click on the ![Add](../images/add_obj.gif?raw=true) *Add* menu item and select the *Create Environment* option. Define the environment name (for example *Pet_Clinic* or any that you like) and click on *Finish*.
+
+You will notice that the created Environment comes with some default settings.  For example several common profiles are already selected:
+ - **[Common] Exclude classes** - defines the default exclude classes patterns. This profile should always be included to insure the correct functioning of the inspectIT agent.
+ - **[Common] HTTP** - defines configuration for instrumenting the HTTP calls. As our application is web based, we will keep this profile included.
+ - **[Commons] Remote-HTTP** - defines configuration for instrumenting remote http calls between different JVMS. As our application consists of several JVMs, we will keep this profile included.
+ - **[Commons] Remote-JMS** - defines configuration for instrumenting remote jms calls between different JVMS. As our application does not use JMS, we can exclude this profile.
+ - **[Commons] SQL** - defines configuration for instrumenting database calls. As our application uses H2 database in the back-end, we will also keep this profile included.
+
+## Profile definition
+### Create new profile
+We need a profile where all our instrumentation points for the monitored application will be defined.  From the ![Instrumentation Manager](../images/compass.png?raw=true) **Instrumentation Manager** view click on the ![Add](../images/add_obj.gif?raw=true) *Add* menu item and select the *Create Profile* option. Define the profile name (for example *Petclinic_profile* or any that you like). For the profile type make sure that you select *Sensor assignment* and click on *Finish*.
+
+### Define instrumentation points
+The new profile does not contain any sensor assignment. We will add several instrumentations points for the Timer sensor, Exception sensor, and Remote Manual Server Sensor:
+
```

Comment:
Ok good idea. I put step 1 to step 3. So the first would be to capture only one method getOwnerDetails()